astroquery/vizier/core.py

```diff
@@ -354,6 +354,9 @@ def query_region_async(self, coordinates, radius=None, inner_radius=None,
         catalog : str or list, optional
             The catalog(s) which must be searched for this identifier.
             If not specified, all matching catalogs will be searched.
+        frame : str, optional
+            The frame to use for the request: can be 'galactic' or 'fk5'.
```

Comment:
there is no actual check that these two are the accepted values, so maybe add a check for it below, or accept any frames that the code later converts to either of these two (so that vizier would understand them, too)

Reply:
Right, the appropriate check is probably:
```python
if frame not in ('galactic', 'fk5'):
    raise ValueError("Only the 'galactic' and 'fk5' frames are supported by Vizier")
```
(though I'm not certain whether we should also allow ICRS?  If we don't know, let's default to excluding it)

Reply:
I think ideally we can allow everything, and if it's not `'galactic'` then we explicitly convert it to icrs and pass it as a "J..." coordinate. 
But your suggestion above should work to too fix the current discrepancy between docs and code.